src/agoradatatools/great_expectations/gx/expectations/team_info.json

```diff
@@ -0,0 +1,208 @@
+{
+  "data_asset_type": null,
+  "expectation_suite_name": "team_info",
+  "expectations": [
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "team",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_not_be_null",
+      "kwargs": {
+        "column": "team"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "team",
+        "max_value": 50,
+        "min_value": 3
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_set",
+      "kwargs": {
+        "column": "team",
+        "value_set": [
+          "ASU",
+          "Harvard-MIT",
+          "Chang Lab",
+          "MSSM - Roussos Lab",
+          "Sage Bionetworks",
+          "IUSM-Purdue",
+          "Longo Lab",
+          "Emory-Sage-SGC",
+          "Emory",
+          "JAX-VUMC-UW Resilience",
+          "Duke",
+          "Duke BARU",
+          "Mayo-UFL-ISB",
+          "Columbia-Rush",
+          "Mayo",
+          "MSSM - Zhang Lab"
+        ]
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_unique",
+      "kwargs": {
+        "column": "team"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "team_full",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_not_be_null",
+      "kwargs": {
+        "column": "team_full"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_value_lengths_to_be_between",
+      "kwargs": {
+        "column": "team_full",
+        "max_value": 100,
+        "min_value": 10
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_regex",
+      "kwargs": {
+        "column": "team_full",
+        "regex": "^[a-zA-Z0-9-() ]+$"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_unique",
+      "kwargs": {
+        "column": "team_full"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_of_type",
+      "kwargs": {
+        "column": "program",
+        "type_": "str"
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_set",
+      "kwargs": {
+        "column": "program",
+        "value_set": [
+          "AMP-AD",
+          null,
```

Comment:
Can we add a multi-field rule to ensure that  if program == null, team == "Sage Bionetworks"? No other team should be allowed to have a null program value...

Reply:
I will take a look at this today

Reply:
@JessterB At first glance, it doesn't look like there is an existing expectation that would do what you are describing. Is this worth creating a custom expectation (~1 sp of work)?

Reply:
@BWMac Eh, not really. We don't add new teams all that often, and when we do there is already enough manual handling and validation required that we really should catch a missing program.  Let's not spend a point on it for now.

Reply:
tracking this via AG-1391 for the future